### PR TITLE
Finish support for LineQubit everywhere.

### LINKED ISSUE #611: Support `LineQubit` and `NamedQubit` in TFQ.
Currently TFQ only supports GridQubit instances in circuits. It would be good to add the functionality for `cirq.LineQubit` as well as `cirq.NamedQubit`. This would require upding our `core/serializer` as well as some surrounding C++ code and proto files. https://github.com/tensorflow/quantum/blob/master/tensorflow_quantum/core/serialize/serializer.py#L826

### PR DESCRIPTION
**Pull request by MichaelBroughton**
Fixes #611 .

### CODE REVIEW

File: tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py [hunk 1]
```diff
@@ -471,7 +471,7 @@ def test_weight_coefficient(self):
 
     def test_simple_pad(self):
         """Test simple padding."""
-        bit = cirq.GridQubit(0, 0)
+        bit = cirq.LineQubit(0)
```
Comment (jaeyoo) [inline tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py:474]: `cirq.LineQubit(1)`

File: tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py [hunk 2]
```diff
@@ -720,7 +720,7 @@ def test_error(self):
 
     def test_many_values(self):
         """Ensure that padding with few symbols and many values works."""
-        bit = cirq.GridQubit(0, 0)
+        bit = cirq.LineQubit(0)
```
Comment (jaeyoo) [inline tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py:723]: `cirq.LineQubit(1)`

**Review by jaeyoo — CHANGES_REQUESTED** [review]
Please look at the minor changes.

File: tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py (hunk 1 above)
Comment (MichaelBroughton) [reply tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py]: Changed.

**Review by jaeyoo — APPROVED** [review]
LGTM

File: tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py (hunk 2 above)
Comment (MichaelBroughton) [reply tensorflow_quantum/core/ops/tfq_ps_util_ops_test.py]: Changed.

**Pull request merged**